annotation-file-utilities/build.gradle

```diff
@@ -53,7 +53,6 @@ dependencies {
     implementation 'org.plumelib:options:1.0.4'
     implementation 'org.plumelib:plume-util:1.1.4'
     implementation 'org.plumelib:reflection-util:0.2.2'
-    compileOnly 'org.checkerframework:checker-qual:3.2.0'
```

Comment:
File `./annotation-file-utilities/src/annotator/find/IsSigMethodCriterion.java` imports `@BinaryName`, which I would have expected comes from here.
Which dependency imports the CF qualifiers now?

Reply:
See #283.